qcore/archive_structure.py

```diff
@@ -0,0 +1,17 @@
+"""
+Gives access to the folder structure of archived cybershake directories
+"""
+from pathlib import Path
+
+from simulation_structure import get_fault_from_realisation
+
+
+def get_IM_csv_from_root(archive_root: Path, realisation: str):
+    """Gets the full path to the im_csv file given the archive root dir and the realistion name"""
+    fault_name = get_fault_from_realisation(realisation)
+    return (
+        archive_root
+        / fault_name
+        / f"{fault_name}_IM"
```

Comment:
We agreed to have the fault_name in .tar, file, eg)  {fault_name}_IM.tar, but when untarred, and recreate the archive directory hierachy, it will be archive_root/ fault_name/ IM, without the  {fault_name} prefix from the IM subdirectory name.